eng/Versions.props

```diff
@@ -6,6 +6,7 @@
     <RestoreSources>$(DotNetRestoreSources)</RestoreSources>
     <RestoreSources Condition="'$(DotNetBuildOffline)' != 'true'">
         $(RestoreSources);
+        https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json;
         https://dotnet.myget.org/F/roslyn/api/v3/index.json;
```

Comment:
nit: surprised to see MyGet is still here.

Reply:
I _think_ we just haven't trimmed this back to what's necessary today. microsoft/msbuild#4426 to follow up on that.